packages/hurumap-core/src/utils/utils.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable import/prefer-default-export */
+export function idify(string) {
+  return string
+    .replace(/^\s+|\s+$/g, "")
```

Comment:
Just to be safe so that we don't exclude uppercase characters,  we did something similar on #795 

```suggestion
export function idify(string) {
  return string
    .toLowerCase()
    .replace(/^\s+|\s+$/g, "")
```

I see with these changes we will have:
[packages/hurumap-core/src/utils/utils.js](https://github.com/CodeForAfrica/ui/pull/793/files#diff-b627c378f09485e15b52d4673b1c81a21799386c142e695689006750eac7b11b) 

Which has an identical function as the one we have in:
[packages/hurumap-core/src/Scope/utils.js](https://raw.githubusercontent.com/CodeForAfrica/ui/main/packages/hurumap-core/src/Scope/utils.js)

Shouldn't we move towards having one `utils.js` shared across? 🤔